pkg/util/cloudfoundry/cloudfoundry.go

```diff
@@ -14,6 +14,8 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/util"
 )
 
+var utilFqdn = util.Fqdn
```

Comment:
We probably want to add a small comment as to why we're doing this :)

nit: wouldn't you prefer a prettier looking name like `getFqdn`?

Reply:
I do :)